esileapclient/tests/unit/osc/v1/test_lease.py

```diff
@@ -169,48 +169,38 @@ def test_lease_list_long(self):
         self.client_mock.lease.list.assert_called_with(filters)
 
         long_collist = [
```

Comment:
Does 'lease list --long' return long fields and 'lease show' return detailed fields?

Reply:
Yep, exactly!